activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -1125,7 +1125,7 @@ def translate_exception_class(native_error, sql, binds)
           active_record_error
         end
 
-        def log(sql, name = "SQL", binds = [], type_casted_binds = [], async: false, &block) # :doc:
```

Comment:
Was removing the block here intentional? Don't we want to pass it to the instrumenter?